index.html

```diff
@@ -767,9 +872,8 @@ <h4>
       denotes the [=digital credential/exchange protocol=].
     </p>
     <p>
-      The {{DigitalCredentialCreateRequest/protocol}} member's value can be one
-      of the well-defined protocol identifiers defined in
-      [[[#protocol-registry]]] or a custom protocol identifier.
+      The {{DigitalCredentialCreateRequest/protocol}} member's value MUST be one
```

Comment:
This basically makes it an enum... if we put MUST here, then we need to also specify what happens when the value is not one of these (i.e., be mindful when this is user agent requirement or an authoring requirement).